doc/nrf/protocols/bt/bt_solutions.rst

```diff
@@ -0,0 +1,82 @@
+Bluetooth solution areas
+########################
+
+Bluetooth technology provides full stack support for a wide and expanding range of use cases.
+
+Audio streaming
+***************
+
+|NCS| supports the Bluetooth LE Audio standard, including Auracast broadcast audio.
+See :ref:`nrf53_audio_app` to get started.
+
+Note that Nordic Semiconductor SoCs do not support Bluetooth Classic Audio, which is based on the
+Bluetooth BR/EDR radio.
+
+Data transfer
+*************
+
+Data transfer is the main use case area for Bluetooth LE and covers use cases like wearables, PC peripherals, and
+health related devices. Supported use cases for Bluetooth LE are implemented as Services and Profiles on top of the
+Generic Attribute Profile (GATT).
+
+Support in |NCS|:
+
+* :ref:`nrf_desktop`: Reference design for a Bluetooth Human Interface Device (HID) solution. Includes desktop mouse, gaming
+  mouse, keyboard, and dongle.
+* :ref:`ble_samples` (|NCS|): Bluetooth samples, most of which are data transfer related. The following
+  samples are recommended to get started with Bluetooth:
+
+  * :ref:`peripheral_lbs`: This sample implements controls for buttons and LEDs on various hardware platforms
+    from Nordic Semiconductor. The GATT Service is not a Bluetooth standard, so the sample is a good template for
+    implementing vendor specific profiles and services.
+
+  * :ref:`peripheral_uart` and :ref:`central_uart` implements a Nordic defined GATT service and profile that give a simple TX/RX
+    generic data pipe, providing UART communication over Bluetooth LE.
+
+* :ref:`bluetooth-samples` (Zephyr Project): The Zephyr Project offers additional Bluetooth samples. These samples are not
+  guaranteed to work as part of |NCS|, but are helpful as starting point for a relvant use case.
+
+Device networks
+***************
+
+Bluetooth Mesh is a technology that enables large-scale device networks in a many-to-many (m:m) topology, with focus
+on reliability and security. The support in |NCS| includes the Mesh Profile, Mesh Models, Bluetooth Networked
+Lighting Control (NLC) profiles, and related application examples. See the :ref:`ug_bt_mesh` main page for further details.
+
+Periodic Advertising with Response (PAwR) allows a central device to communicate with 1000s of devices in a star
+topology. The feature is support in SoftDevice Controller, and the related Encrypted Advertising (EAD) feature is
+supported in Zephyr Host. The main use case, Electronic Shelf-Labels (ESL), is implemented as an application here
+(experimental state): https://github.com/NordicPlayground/nrf-esl-bluetooth
```

Comment:
I am hoping this ESL application will be merged with NCS, in which case it will be an internal link. Could we leave it like this for now to avoid cleaning up links.txt later?

Reply:
Unless the application is going to be merged in the next few days, it's still better to have the link in the `links.txt`.